src/Analysis/Ast/Impl/Specializations/Typing/Types/GenericTypeParameter.cs

```diff
@@ -77,6 +83,22 @@ public GenericTypeParameter(string name, IPythonModule declaringModule, IReadOnl
             return true;
         }
 
+        /// <summary>
+        /// Given arguments to TypeVar, finds the bound type
+        /// </summary>
+        private static IPythonType GetBound(IArgumentSet argSet) {
+            var eval = argSet.Eval;
+            var rawBound = argSet.GetArgumentValue<object>("bound");
```

Comment:
If it can be a variable, then you may want `t = rawBound.GetPythonType()`

Reply:
Sure